src/redirects/redirects.ts

```diff
@@ -1,42 +1,20 @@
+/* eslint-disable no-console */
 import { TRedirect } from "@types";
 
-// for pages that are not in cclw's sitemap
-const cclwRedirects = [
-  { source: "/cookie-policy", destination: "/", permanent: true },
-  { source: "/privacy", destination: "/", permanent: true },
-];
-
-// for pages that are not in cpr's sitemap
-const cprRedirects = [
-  { source: "/about", destination: "/", permanent: true },
-  { source: "/acknowledgements", destination: "/", permanent: true },
-  { source: "/contact", destination: "/", permanent: true },
-  { source: "/methodology", destination: "/", permanent: true },
-  { source: "/faq", destination: "/", permanent: true },
-];
-
-const theme = process.env.THEME;
+// Please note: this file is no longer being importing into the middleware
 
 function getRedirects(): Array<TRedirect> {
-  const REDIRECT_FILE = process.env.REDIRECT_FILE || "CCLW_redirects.json";
+  const REDIRECT_FILE = process.env.REDIRECT_FILE || `../../themes/${process.env.THEME}/redirects.json`;
```

Comment:
This default is at odds with the way the code is written in other places ... there the path is absolute, e.g. `/themes/${process.env.THEME}/layouts/main` and not relative. I prefer the absolute as it assumes we are running in a container and IMO is safer :)

Reply:
Yes I agree - I will have another look at this but I feel like I tried this first. I'll also look at using a path alias in the tsconfig

Reply:
Ah - this file is dead now anyway 😆 it has been superseded by the theme-level configuration in the `next.config.js` file